xbmc/utils/SystemInfo.cpp

```diff
@@ -873,6 +877,8 @@ CSysInfo::WindowsVersion CSysInfo::GetWindowsVersion()
         m_WinVer = WindowsVersionWin10_1909;
       else if (osvi.dwMajorVersion == 10 && osvi.dwMinorVersion == 0 && osvi.dwBuildNumber == 19041)
         m_WinVer = WindowsVersionWin10_2004;
+      else if (osvi.dwMajorVersion == 10 && osvi.dwMinorVersion == 0 && osvi.dwBuildNumber >= 22000)
```

Comment:
Have they really just kept the major as 10? Or are the previews all just running with that for now?

Reply:
It seems a bit strange but at the same time it is logical that they do it like this to avoid compatibility problems. That is, v10.0.xxxxx continues to be reported so that they believe that it is still Windows 10 (in fact it is another feature update similar to the previous ones but they have decided to give it another name). With this they prevent potential "bad coded" applications (and probably unmaintained) from having unexpected behavior/even crash if they read a version number "11.0" that did not exist until now.

Windows 11 is in the RC/RTM phase already and I don't think it will change any version number. Even the build number will still be 22000 at launch:

v10.0.22000.yyyy
yyyy = UBR (Update Build Revision)

Reply:
All good. Apple did a similar thing during there pre release stuff, but eventually bumped to 11 when it was formally released

Reply:
They fake the version number unless we specify a guid in the application manifest. Afaik they have not yet released the new guid so we cannot get it to return 11. Maybe they'll keep using 10 this time or we'll get the new guid in October with the release and the new sdk